src/Components/List/List.tsx

```diff
@@ -16,6 +23,25 @@ export interface Props {
 
 export function List(props: Props) {
   const { title, list, selectedChatRoom } = props;
+  const [modalVisible, setModalVisible] = useState(false);
```

Comment:
Don't you think this code is going to be repeated a lot of time? whenever we will use Modal we will have to use this. Can we create a custom hook called `useVisibilityToogle`. Which will expose 2 functions: `show` and `hide` plus 1 variable `isVisible`

Reply:
code repeatation. We have already created a hook for this